### ISSUE TITLE
[FR] <x-rds.Lookup-Secret Iam access> 

### ISSUE THREAD

**Issue by MuyiwaOla** [compose-x/ecs_composex#642]
**Is your feature request related to a problem? Please describe.**
I was unable to pass secret access to the service using x-rds lookup.

**Describe the solution you'd like**
x-rds.Lookup to grant IAM access to the secret.

**Comment by JohnPreston** [comment]
Will be released in 0.23 officially (and documented).
Here is what it will be like

```yaml
x-rds:
  dbA:
    Services:
      app01:
        Access: RW
      app03:
        Access: RW
        GrantTaskAccess:
          SecretEnvName: DB_B_SECRET # Grants access to the secret, sets env var DB_B_SECRET to the ARN
      youtoo:
        Access: RW
        GrantTaskAccess: True # Grants access to the secret, not setting an env var
```

**Comment by JohnPreston** [comment]
Fixed in #650
